cmake/DLAF_AddTest.cmake

```diff
@@ -8,21 +8,125 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-# DLAF_addTest(test_target_name
-#   SOURCES <source1> [<source2> ...]
-#   [COMPILE_DEFINITIONS <arguments for target_compile_definitions>]
-#   [INCLUDE_DIRS <arguments for target_include_directories>]
-#   [LIBRARIES <arguments for target_link_libraries>]
+set(DLAF_PIKATEST_EXTRA_ARGS "" CACHE STRING "Extra arguments for tests with pika")
+
+macro(dlaf_setup_mpi_preset)
+  # if a preset has been selected and it has been changed from previous configurations
+  if(NOT DLAF_MPI_PRESET STREQUAL _DLAF_MPI_PRESET)
+
+    if(DLAF_MPI_PRESET STREQUAL "plain-mpi")
+      # set(MPIEXEC_EXECUTABLE "" CACHE STRING "Executable for running MPI programs")
+      # set(MPIEXEC_NUMPROC_FLAG "" CACHE STRING "Flag used by MPI to specify the number of processes for mpiexec; the next option will be the number of processes." FORCE)
```

Comment:
```suggestion
```
Remove commented out code?

Reply:
I'm ok with that, it was just a way of "self-documenting' that for this preset that's the "default" and how it gets configured.

Let me know if you still want to delete it or not 😉